features/clipboard-svg.yml

```
@@ -0,0 +1,6 @@
+name: Async clipboard support for SVG
+description: Support for the `image/svg+xml` mime-type in the `navigator.clipboard` API allows you to read and write SVG data in the system clipboard.
```

Avoiding "support for" because that's the kind of phrasing we could put in every single feature, if we let it become a habit. 😄 

```suggestion
name: SVG clipboard items
description: The `image/svg+xml` data type (also known as the MIME type) for `ClipboardItem` objects represent SVG data written to or read from the clipboard.
```

---

```suggestion
name: SVG clipboard items
description: The `image/svg+xml` data type (also known as the MIME type) for `ClipboardItem` objects represents SVG data written to or read from the clipboard.
```